src/components/SchemaRow/SchemaRow.tsx

```diff
@@ -52,89 +47,94 @@ export const SchemaRow: React.FunctionComponent<SchemaRowProps> = ({ schemaNode,
 
   const isBrokenRef = typeof refNode?.error === 'string';
 
+  const rootLevel = renderRootTreeLines ? 1 : 2;
   const childNodes = React.useMemo(() => calculateChildrenToShow(typeToShow), [typeToShow]);
   const combiner = isRegularNode(schemaNode) && schemaNode.combiners?.length ? schemaNode.combiners[0] : null;
+  const isCollapsible = childNodes.length > 0;
+  const isRootLevel = nestingLevel < rootLevel;
```

Comment:
I don't know if it matters, but should this be `<=`?

Reply:
Not here actually - yeah it's kind of confusing 🤔

Reply:
That's fine I just wasn't sure if `nestingLevel === rootLevel` counted as `isRootLevel` or not. 😄